### PR TITLE
Ember cli/es6 port

### PR DESCRIPTION
**Pull request by miguelcobain**
Ignore #88.
This uses the orphan branch @codefox421 created.

Original issue #68

Travis build is passing, and dummy app renders the default map.

It's a start!

### CODE REVIEW

File: app/layers/marker-collection.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+/**
+  `EmberLeaflet.MarkerCollectionLayer` is a specific collection layer
+  containing marker objects.
+
+  @class MarkerCollectionLayer
+  @namespace EmberLeaflet
+  @extends EmberLeaflet.CollectionLayer
+  @uses EmberLeaflet.BoundsMixin
+*/
+export default CollectionLayer.extend({
```
Comment (miguelcobain) [inline app/layers/marker-collection.js]: Beautiful.

**Comment by twokul** [comment]
@gabesmed any movement on this one? I would love for this one to land! @miguelcobain awesome work dude

**Comment by gabesmed** [comment]
Whoa. @miguelcobain you ported the whole thing?!?!?!

**Comment by miguelcobain** [comment]
Not yet. Geometry layers and lots of tests are missing.

I was hoping to get some help, it should be easy from now. :)
Also, I was hoping to see this merged. :D

**Comment by gabesmed** [comment]
MERGED :D

**Comment by gabesmed** [comment]
sorry been out of communications but this is frickin amazing thanks @miguelcobain !!!!!! :100:

**Comment by lilfaf** [comment]
Hi guys!
I'm having a hard time importing @miguelcobain branch after installing through npm.
`npm install miguelcobain/ember-leaflet#ember-cli-es6 --save-dev`
`ember g ember-leaflet`
This error keeps showing `Could not find module ember-leaflet/components/ember-leaflet`
Is there something i'm missing ? (noob ember-cli user here)

**Pull request merged**